lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenProjectCache.java

```diff
@@ -119,9 +122,18 @@ private void check(DOMDocument document) {
 			}
 		}
 		ProjectBuildingRequest request = new DefaultProjectBuildingRequest();
-		request.setSystemProperties(System.getProperties());
+		Properties systemProperties = new Properties();
+		EnvironmentUtils.addEnvVars(systemProperties);
```

Comment:
This is the actual fix.

Reply:
Couldn't/shouldn't this properties be initialized only once in constructor rather than on every request? I don't think either the system.getProperties() nor the environment can nor should change during execution of the LS (in worst case, people would just have to restart it), so it may be good to store it once and for all.

Reply:
+1, done